src/tactic/ui/panel/hash_panel_wdg.py

```diff
@@ -162,11 +162,34 @@ def _get_predefined_url(cls, key, hash):
                 # put in a check to ensure this is a user
                 parts = link.split(".")
 
+                '''For usernames with at most one period
                 # you can only see your personal links
                 user = Environment.get_user_name()
                 if user == parts[0]:
                     personal = True
+                elif parts[1]:
```

Comment:
oh ok. Now I see you were adding codes but the whole block get commented out.. It's better to turn those python lines into a brief description instead.. since is_personal() is taking care of all cases already.